deploy/01_devSetup.ts

```diff
@@ -27,7 +27,7 @@ async function deployDevToken(
 
   const isDeployed = await getOrNull(name);
 
-  if (isDeployed) {
+  if (isDeployed && !includes(['hardhat', 'docker'], getNetworkName())) {
```

Comment:
Don't we already have a skip for these?

Reply:
yeah but notice goerli is not included here, like that the tokens get verified when deployed to goerli